js/settings-personal.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * @author Piotr Mrowczynski <piotr@owncloud.com>
+ *
+ * @copyright Copyright (c) 2023, ownCloud GmbH
+ * @license AGPL-3.0
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+(function ($, OC, OCA) {
+
+	OCA.RichdocumentsPersonalSettings = {
```

Comment:
It seems better to split the namespace into `OCA.Richdocuments.Settings.Personal`. At least I expect the name of the app to be after the `OCA` prefix. Whether the rest is `Settings.Personal`, `SettingsPersonal` or `PersonalSettings` is likely irrelevant and it might depend on future plans (I don't think we'll need to add too much js code into different namespaces.)

Reply:
done, makes sense